labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,98 @@
+
+## Current Features
+
+### PLATFORM SUPPORT
+
+- Server
+- Kubernetes
+- Docker
+
+### AGENT TYPES
+
+- Issuer
+- Holder
+- Verifier
+
+### CREDENTIAL TYPES SUPPORTED
+
+- W3C JWT-VC
+- Anoncreds
+
+### DID METHODS SUPPORTED
+
+- did:prism
+- did:peer
+- Other did-methods can be verified via a universal resolver
+
+### COMMUNICATION SUPPORT
+
+- DIDCOMM V2
+
+### DIDCOMM PROTOCOLS SUPPORTED
+
+- [Mediator Coordinator](https://didcomm.org/mediator-coordination/2.0/)
+- [DIDComm V2 Messaging](https://identity.foundation/didcomm-messaging/spec)
+- [DIDComm V2 Issue Credential](https://github.com/decentralized-identity/waci-didcomm/tree/main/issue_credential)
+- [DIDComm V2 Present Proof](https://github.com/decentralized-identity/waci-didcomm/blob/main/present_proof/present-proof-v3.md)
+- [DIDComm V2 Report Problem](https://identity.foundation/didcomm-messaging/spec/#problem-reports)
+- [DIDComm V2 Routing Protocol](https://identity.foundation/didcomm-messaging/spec/#routing-protocol-20)
+
+### SUPPORTED ARIES RFC’s
+
+- [0023-did-exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange)
+- [0434-out-of-band-protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md)
+- [0453-issue-credential-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2)
+- [0453-present-proof-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0454-present-proof-v2)
+
+# Initial Committers
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/FabioPinheiro
+- https://github.com/mineme0110
+- https://github.com/bvoiturier
+- https://github.com/yshyn-iohk
+- https://github.com/CryptoKnightIOG
+- https://github.com/shotexa
+- https://github.com/goncalo-frade-iohk
+- https://github.com/lohanspies
+
+# Initial Maintainers
+
+- https://github.com/bsandmann
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/yshyn-iohk
+
+# Sponsors
+- Lance Byrd - ([lance.byrd@rootsid.com](mailto:lance.byrd@rootsid.com)) - Aries DIDCommV2 WG member
+- Stephen Curran (swcurran@cloudcompass.ca) - ACA-Py Maintainer, Aries RFC Maintainer
+- Timo Glastra - ([timo@animo.id](mailto:timo@animo.id)) - Aries Framework JavaScript Maintainer, ACA-Py Contributor
+- Daniel Hardman - ([daniel.hardman@gmail.com](mailto:daniel.hardman@gmail.com)) - Lead of ToIP Trust Spanning Task Force, Primary editor ToIP ACDC Task Force, Aries RFC Maintainer
+- Darrell O’Donnell - ([darrell.odonnell@continuumloop.com](mailto:darrell.odonnell@continuumloop.com)) - Chair of ToIP Technology Stack WG, Chair of ToIP Technology Architecture Task Force & Lead of ToIP Trust Registry Task Force
+
+# Pre-existing repository
```

Comment:
I would like to understand how this repository is getting used in this lab. 
Could you provide me access to [this](https://github.com/input-output-hk/atala-prism-building-blocks) repository.

Reply:
@knagware9 Not sure if there's a misunderstanding, but the https://github.com/input-output-hk/prism-building-blocks  repo is the one we intend to transfer to Hyperledger. What we wanted to clarify is that we are making use of a library which is GNU LGPL Licensed. I don't think that's a problem, but I'm not a legal expert.

Reply:
Okay